tests/ProjectsTest.php

```diff
@@ -184,6 +185,53 @@ public function testUsersManagement()
         $this->assertCount(1, $admins);
     }
 
+    public function testTemporaryAccess()
+    {
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
+        ]);
+
+        $project = $this->client->createProject($organization['id'], [
+            'name' => 'My test',
+        ]);
+        $admins = $this->client->listProjectUsers($project['id']);
+        $this->assertCount(1, $admins);
+
+        // test of adding / removing user with expiration/reason
+        $resp = $this->client->addUserToProject($project['id'], [
+            'email' => 'spam@keboola.com',
+            'reason' => 'created by test',
+            'expires' => date("Y-m-d H:i:s", strtotime("+ 1 minute"))
```

Comment:
@pivnicek I think it would be better to change this parameter to `expirationSeconds`, it will be safer for timezone issues and more unified with similar api calls like http://docs.keboolamanagementapi.apiary.io/#reference/projects/delete-project/update-project